src/main/config/config.c

```diff
@@ -104,7 +104,7 @@ pidProfile_t *currentPidProfile;
 #define DYNAMIC_FILTER_MAX_SUPPORTED_LOOP_TIME HZ_TO_INTERVAL_US(2000)
 
 #define BETAFLIGHT_MAX_SRATE  100
-#define KISS_MAX_SRATE        100
+#define KISS_MAX_SRATE        99
```

Comment:
why does this have to be 99?

Reply:
https://github.com/betaflight/betaflight/blob/c6d70214f54121e84ddbd2d90bcecff7366a1df7/src/main/fc/rc.c#L190
Any value over 99 would just give the same result as 99. 
In some versions of this function ```kissRpyUseRates``` isn't constrained so I would guess it has to be 99 to avoid divide by 0. Here for example https://github.com/apocolipse/RotorPirates/blob/master/RotorPirates.py.

Reply:
@klutvott123 yeah now I remember how it works (seems that I wrote it 😆)

Reply:
Also the Configurator limits it to 99. So the main point is that the behavior in the firmware and Configurator should be consistent.